eZ/Publish/Core/settings/search_engines/legacy/criterion_handlers_common.yml

```diff
@@ -65,6 +65,8 @@ services:
     ezpublish.search.legacy.gateway.criterion_handler.common.content_type_identifier:
         parent: ezpublish.search.legacy.gateway.criterion_handler.base
         class: %ezpublish.search.legacy.gateway.criterion_handler.common.content_type_identifier.class%
+        arguments:
+            - @ezpublish.spi.persistence.content_type_handler
```

Comment:
True... fixed in https://github.com/ezsystems/ezpublish-kernel/commit/1279bebe6c8fd7cac5132c87aa76a532fcc8e94f, thanks.